pkg/controllers/configobservation/console/observe_consoleurl.go

```diff
@@ -20,8 +23,24 @@ func ObserveConsoleURL(genericlisters configobserver.Listers, recorder events.Re
 	listers := genericlisters.(configobservation.Listers)
 	errs := []error{}
 
+	clusterVersionConfig, err := listers.ClusterVersionLister.Get("version")
+	if err != nil {
+		return existingConfig, append(errs, err)
+	}
+
+	isConsoleCapabilityEnabled := false
+	for _, capability := range clusterVersionConfig.Status.Capabilities.EnabledCapabilities {
+		if capability == configv1.ClusterVersionCapabilityConsole {
+			isConsoleCapabilityEnabled = true
```

Comment:
shouldn't the execution be stopped with a hardcoded value once the capability is not allowed?

Reply:
it should 👍